### DISCUSSION TITLE
Mock `performance.now()` for edge presets

### DISCUSSION THREAD

**Discussion by leo91000** [nitrojs/nitro#1681] [Q&A]
Hello,

My project can't deploy to edge because the compiled edge output has `performance.now` references wich edge runtime doesn't support.

Here is an example error message I get from vercel logs once deployed :

```shell
[nuxt] [request error] [unhandled] [500] Cannot read properties of undefined (reading 'now')
  at (index.mjs:1:116679)  
  at (index.mjs:1:116786)  
  at (index.mjs:1:117336)  
  at (index.mjs:1:118554)  
  at (index.mjs:1:122217)  
  at (index.mjs:1:66208)  
  at (index.mjs:1:68367)  
  at (index.mjs:1:145413)  
  at (index.mjs:1:115455)  
  at (index.mjs:1:68471)
```

I have 2 questions :
- Is it possible to have a global mock for performance.now ? if so how can I do it 
- Is there a way to find out which library this output comes from ? I'm suspecting vuetify, but I can't be sure

Thanks

**Comment by pi0** [comment]
Hi. Sure it would be possible. Would you please open a feature request in https://github.com/unjs/unenv? 🙏🏼

**Reply by leo91000** [reply]
I opened a feature request here : [https://github.com/unjs/unenv/issues/137](https://github.com/unjs/unenv/issues/137#issue-1879172987)
